src/amber/environment/loader.cr

```diff
@@ -1,15 +1,11 @@
-require "./settings"
-
-module Amber
-  class EnvironmentLoader
-    # TODO: Should probably use Amber::Env type here but can't until we refactor the Amber module.
-    # Should default path to Amber.environment_path
-    def initialize(@environment : String | Symbol = :development, @path : String = "./config/environments")
+module Amber::Environment
+  class Loader
+    def initialize(@environment : Amber::Environment::EnvType, @path : String)
```

Comment:
Could we default these to `:development` and `@@environment_path` or something?

Reply:
This class should not make assumptions about the environment and the path. To ensure The default values are set  to the _Amber.environment_path_ property and **Amber.env.to_s**

Reply:
Defaults are fine as long as they can be overwritten, IMO.